### ISSUE TITLE
[JsonPropertyAttribute("objectID")] not working correctly

### ISSUE THREAD

**Issue by kevinp** [algolia/algoliasearch-client-csharp#539]
We hit a similar bug to Issue #536 , but we're not using a base class with our POCO.  The following decoration results in the exception: "The type T must have an ObjectID property or a JsonPropertyAttribute with name='objectID'"

```
        [JsonPropertyAttribute("objectID")]
        public string ObjectId { get; set; } 
```
I've reviewed the code, and can't figure out why this always fails:

if (typeInfo.GetCustomAttribute<JsonPropertyAttribute>() != null)

**Comment by Ant-hem** [comment]
Hey!

Thanks for your contribution. I have fixed this issue with 2a150b4 in #537.
I will publish a patch version 6.0.1 by the end of the day that is fixing this issue.

Thanks!

**Comment by Ant-hem** [comment]
Solved with [6.0.1](https://www.nuget.org/packages/Algolia.Search/6.0.1)
